diff --git a/node_modules/@vendure/dashboard/src/app/routes/_authenticated.tsx b/node_modules/@vendure/dashboard/src/app/routes/_authenticated.tsx
index 34ed122..68c4267 100644
--- a/node_modules/@vendure/dashboard/src/app/routes/_authenticated.tsx
+++ b/node_modules/@vendure/dashboard/src/app/routes/_authenticated.tsx
@@ -1,9 +1,8 @@
 import { AppLayout } from '@/vdb/components/layout/app-layout.js';
-import { AUTHENTICATED_ROUTE_PREFIX } from '@/vdb/constants.js';
 import { useAuth } from '@/vdb/hooks/use-auth.js';
 import { createFileRoute, redirect, useNavigate } from '@tanstack/react-router';
 
-export const Route = createFileRoute(AUTHENTICATED_ROUTE_PREFIX)({
+export const Route = createFileRoute('/_authenticated')({
     beforeLoad: ({ context, location }) => {
         if (!context.auth.isAuthenticated) {
             throw redirect({
